sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -423,3 +430,28 @@ def list_available_area_codes(
                 "administrative_division", None),
             **kwargs
         )
+
+    @distributed_trace
+    def search_operator_information(
+            self,
+            phone_numbers,  # type: PhoneNumberSearchType
+            options=None, #type: OperatorInformationRequestOptions
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> OperatorInformationResult
```

Comment:
for newer additions we should be using the python 3 typing conventions, so we can remove this line and type the function.